src/app/core/filter/filter/filter.component.spec.ts

```diff
@@ -58,38 +62,57 @@ describe("FilterComponent", () => {
   });
 
   it("should support a boolean filter", async () => {
```

Comment:
I think the correct place for testing the actual filter functionality would be in `filters.spec.ts`